app/models/metric/statistic.rb

```diff
@@ -0,0 +1,12 @@
+module Metric::Statistic
+  def self.calculate_stat_columns(obj, timestamp)
+    return {} unless obj.respond_to?(:container_groups)
+
+    capture_interval = Metric::Helper.get_time_interval(obj, timestamp)
+
+    {
+      :stat_containergroup_create_rate => obj.container_groups.where(:created_on => capture_interval).count,
+      :stat_containergroup_delete_rate => obj.container_groups.where(:deleted_on => capture_interval).count
```

Comment:
@simon3z It's the provider user is querying about, will ask @zeari how to query for this data.